CreateJobs.groovy

```diff
@@ -8,6 +8,8 @@ def repositories = [
   'esn-frontend-common-libs',
   'esn-frontend-contacts',
   'esn-frontend-inbox',
+  'esn-frontend-login',
+  'esn-frontend-calendar-public',
```

Comment:
Please take care of alphabetic ordering.

Reply:
thanks, I updated the PR